webpack.config.js

```diff
@@ -97,9 +103,20 @@ module.exports = {
     new CopyWebpackPlugin({
       patterns: [{ from: 'assets' }],
     }),
+    new webpack.HotModuleReplacementPlugin(),
+    new ReactRefreshWebpackPlugin({
+      overlay: false,
+    }),
   ],
   externals: {
     // eslint-disable-next-line global-require
     fs: require('fs'),
   },
 };
+
+if (!isDevMode) {
+  // The bom plugin breaks the dev server and hot
+  webpackConfig.plugins.push(new BomPlugin(true));
```

Comment:
@murilovmachado I'm not sure what problem this plugin was solving and if it is still a problem. It looked like it came in with a change that you made a couple of years ago. It breaks hot reload so I turned it off for dev mode. Just want to make sure that won't cause any issues.